Extensions/panaroma.js

```diff
@@ -42,13 +42,14 @@ XKit.extensions.panaroma = new Object({
 		XKit.extensions.panaroma.resized();
 	},
 
+	//added fixes for Spotlight a.k.a. "Staff Picks" page and removed obsolete fixes for Ignore page
 	do_directory_fixes: function() {
 
 		var m_css = " .l-content { padding-bottom: 30px!important; border-radius: 20px!important; background: white!important; } .content_top, .content_bottom { display: none!important; } #tabs { background: #eaeaea!important; } #tabs.tabs_3 .tab { width: 33%!important; } #tabs.tabs_3 .tab:last-child { width: 32%!important; } ";
 
-		if (document.location.href.indexOf("://www.tumblr.com/ignore") !== -1) {
-			m_css = m_css + " #left_column { width: 100%!important; } #content { padding-top: 30px!important;  } ";
-		}
+		if (document.location.href.indexOf("://www.tumblr.com/spotlight") !== -1) {
+    			m_css = m_css + " .chrome_nav { width: 24%!important; min-width: 230px!important; } #cards { width: 75%!important; min-width: 650px!important; } #cards .card{ width:30%!important; min-width:190px!important; }#content { padding-top: 30px!important;  } ";
```

Comment:
@bvtsang generally speaking, where we know we're overriding tumblr styles its probably best to put !important even if the overrides work now—it future-proofs us from tumblr changing selectors in the future, and nudging us out by a little bit. (i.e. blacklist was broken for a long time because tumblr changed a .post selector to a .post.post-full selector, which gave it precedence over our .post selector)